src/zephyr/mercury.ts

```diff
@@ -5,23 +5,30 @@ export const getMercuryInstance = (network: "MAINNET" | "TESTNET") => {
   let graphqlEndpoint = null;
   let email = null;
   let password = null;
+  let apiKey = null;
+  let jwt = null;
 
   if (network === "MAINNET") {
     email = process.env.MERCURY_EMAIL_MAINNET!;
     password = process.env.MERCURY_PASSWORD_MAINNET!;
     backendEndpoint = process.env.MERCURY_BACKEND_ENDPOINT_MAINNET!;
     graphqlEndpoint = process.env.MERCURY_GRAPHQL_ENDPOINT_MAINNET!;
+    apiKey = process.env.MERCURY_API_KEY_MAINNET!;
+    jwt = process.env.MERCURY_JWT_MAINNET!;
   } else {
     email = process.env.MERCURY_EMAIL_TESTNET!;
     password = process.env.MERCURY_PASSWORD_TESTNET!;
     backendEndpoint = process.env.MERCURY_BACKEND_ENDPOINT_TESTNET!;
     graphqlEndpoint = process.env.MERCURY_GRAPHQL_ENDPOINT_TESTNET!;
+    apiKey = process.env.MERCURY_API_KEY_TESTNET!;
+    jwt = process.env.MERCURY_JWT_TESTNET!;
   }
 
   return new Mercury({
     backendEndpoint,
     graphqlEndpoint,
-    email,
-    password,
+    apiKey,
```

Comment:
If we're using `shouldFetchApiKey` and also passing the `jwt` then I don't think we need to pass the `apiKey` to the mercury instance since it's already generating it.

Reply:
You're right, I forget to remove it, will do a new commit